feat: Adding KTX for Maps SDK V3 Beta (#51)
googlemaps/android-maps-ktx: maps-v3-ktx/README.md

maps-v3-ktx
===========

This module contains Kotlin extensions (KTX) for the Maps SDK V3 BETA library for Android. 

> **NOTE**: The source code in this module is autogenerated from source code within the `maps-ktx` 
> and is not intended for modifications. If any edits should be made, they should be made in that 
> module first.

Reviewer: I would add when/how the code is generated:

```suggestion
**NOTE**: The source code in this module is autogenerated from source code within the `maps-ktx` 
module (on `maps-ktx` compilation) and is not intended for modifications. Please make any code 
changes in the `maps-ktx` module.
```

Author: Good call 💯
